src/pages/overview/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { RouteObject, Outlet } from "react-router-dom";
+import React, { lazy, Suspense } from "react";
+import { CircularLoader } from "@dhis2/ui";
+import { DefaultLazyLoad } from "../../components";
+
+// React.lazy only works with default exports, so we import directly instead of from index.tsx
+const AllOverview = lazy(() => import("./AllOverview"));
+const DataElements = lazy(() => import("./DataElements"));
+
+const OverviewWrapper = () => <Outlet />;
+
+const route: RouteObject = {
```

Comment:
Index files' purpose should be to define the public interface of a module (folders in the case of JS). But this file also defines routes. If we decide to keep route definitions in the pages folder (which I'm against), then this should go into its own file.

Reply:
https://dhis2.atlassian.net/browse/DHIS2-15077